[ICDS] - changes

lwyszomi opened the pull request:
Hi, 

In the commits I added changes for two cases: 
https://manage.dimagi.com/default.asp?267436 - someone removed Tableau report and forgot to add a comma after Dashboard report.
https://manage.dimagi.com/default.asp?266735#1436546 - I changed to 'India format' all numbers which required this

Regards,
Łukasz

Commit:
Use india number format

Commit:
Added comma in the custom report configuration

Commit:
stickler

czue commented on custom/icds_reports/utils.py:
```diff
@@ -457,3 +458,14 @@ def create_pdf_file(pdf_hash, pdf_context):
+        amount, group = divmod(amount, 100)
+        groups.insert(0, group)
+
+    return ','.join(map(str, groups))
```
no big deal but this is the type of function that would be nice to have a couple unit tests for to document it and make sure it's working right. also it looks like it will fail hard on invalid inputs - confirming that's desired?

czue reviewed (approved):
Looks good but had one comment re: tests.

:india:

Commit:
refactor india numbers and add tests

lwyszomi commented:
@czue I changed the implementation of the ```indian_formatted_number``` because when I added tests I noticed that the previous implementation didn't work properly. Please let me know if you have any additional questions

Commit:
stickler

Commit:
thicker line for <hr> element

czue commented:
great that the tests caught a bug! :raised_hands:
